src/index.html

```diff
@@ -19,9 +19,389 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#home"
+      >
+        <img
+          class="image-logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```